doc/requirements.txt

```diff
@@ -0,0 +1,3 @@
+sphinx >= 4.5.0
+sphinx_rtd_theme >= 1.0.0
+sphinx-copybutton >= 0.5.0
```

Comment:
Could you point the "requirements" in the doc makefile job here too?

Reply:
Done!